scripts/local-universe.py

```diff
@@ -278,8 +278,19 @@ def selected_check(package_json, packages, only_selected, is_latest):
 
 
 def load_json(json_path):
-    with json_path.open(encoding='utf-8') as json_file:
-        return json.load(json_file)
+    try:
+        with json_path.open(encoding='utf-8') as json_file:
+            return json.load(json_file)
+    except json.JSONDecodeError as err:
```

Comment:
How about : 
```
except json.JSONDecodeError as err:
    print("JSON error in file: %s" % json_path)
    print_width = 50
    snippet = err.doc[max(0, err.pos - print_width):max(0, err.pos + print_width)]
    if err.pos - print_width > 0:
        snippet = '... %s' % snippet
    if err.pos + print_width < len(err.doc):
        snippet = '%s ...' % snippet
    print(err)
    print(snippet)
```
- This doesn't have temporary variables `start, stop`.
- The appending/prepending of `...` is consistent in style
- Just use one random number value of `50` instead of `20, 40, 60`. Makes the code more readable.

Reply:
Sure, I agree it's more readable. I've rebased and updated the file to your version.